WordPressCom-Stats-iOS/InsightsTableViewController.m

```diff
@@ -206,8 +417,80 @@ - (NSString *)cellIdentifierForIndexPath:(NSIndexPath *)indexPath
             }
             break;
 
-        case StatsSectionGraph:
         case StatsSectionPeriodHeader:
+            return StatsTablePeriodHeaderCellIdentifier;
+
+        case StatsSectionTagsCategories:
+        case StatsSectionPublicize:
+        {
+            StatsGroup *group = (StatsGroup *)[self statsDataForStatsSection:statsSection];
+            if (indexPath.row == 0) {
+                identifier = StatsTableGroupHeaderCellIdentifier;
+            } else if (indexPath.row == 1 && group.numberOfRows > 0) {
+                identifier = StatsTableTwoColumnHeaderCellIdentifier;
+            } else if (indexPath.row == 1) {
+                identifier = StatsTableNoResultsCellIdentifier;
+            } else if (group.moreItemsExist && indexPath.row == (NSInteger)(group.numberOfRows + StatsTableRowDataOffsetStandard)) {
```

Comment:
I have a bunch of compiler warnings turned up. One of those is the implicit casting warning.